contents/docs/integrate/_snippets/install-nuxt.mdx

```diff
@@ -1,3 +1,5 @@
+<!-- Keep in sync with https://github.com/PostHog/posthog-js/tree/main/playground/nuxtjs -->
```

Comment:
the mdx docs suggest I can use this comment syntax but I'm not 100% confident, waiting for the preview deploy